doc/rst/fabrics.rst

```diff
@@ -92,6 +93,9 @@ Fabrics-specific definitions.
 ``tls``
   Start TLS on the connection (TCP)
 
+``concat``
```

Comment:
The `doc/rst/fabtrics.rst` file gets auto generated. No need to add it to the change.